app/models/profile.rb

```diff
@@ -24,6 +24,12 @@ class Profile < ApplicationRecord
   delegate :email, :role, to: :user
 
   scope :only_tutors, (-> { where(users: { role: :tutor }) })
+  scope :search_profiles, (lambda do |search_query|
+    return if search_query.blank?
```

Comment:
Let's also `return all if search_query.blank?` and remove `all` call from controller